match/coms/service_coms_handler.go

```diff
@@ -0,0 +1,30 @@
+package coms
+
+import (
+	"github.com/TempleEight/spec-golang/match/utils"
+	"fmt"
+	"net/http"
+)
+
+// Handler maintaints the list of services and their associated hostnames
+type Handler struct {
+	Services map[string]string
+}
+
+// Initialise sets up the Handler object with a list of services from the config
+func (coms *Handler) Initialise(config *utils.Config) {
```

Comment:
British English:
![](https://media.giphy.com/media/Nszyj17J4fUKmIwQwF/giphy.gif)

More to the point, is that what we want our generated code to be in?